File: backend/src/BiteRight.Application/Queries/Products/Search/SearchResponse.cs
// # ==============================================================================
// # Solution: BiteRight
// # File: ListResponse.cs
// # Author: Łukasz Sobczak
// # Created: 16-02-2024
// # ==============================================================================

#region

using BiteRight.Application.Dtos.Common;
using BiteRight.Application.Dtos.Products;

#endregion

namespace BiteRight.Application.Queries.Products.Search;

public record SearchResponse(PaginatedList<SimpleProductDto> Products);

File: mobile/app/src/main/java/com/sobczal2/biteright/data/api/responses/products/SearchResponse.kt
package com.sobczal2.biteright.data.api.responses.products

import com.google.gson.annotations.SerializedName
import com.sobczal2.biteright.dto.common.PaginatedList
import com.sobczal2.biteright.dto.products.SimpleProductDto

data class SearchResponse(
    @SerializedName("products") val products: PaginatedList<SimpleProductDto>
)